lxd/storage_volumes.go

```diff
@@ -2728,6 +2728,84 @@ func createStoragePoolVolumeFromBackup(s *state.State, r *http.Request, requestP
 	return operations.OperationResponse(op)
 }
 
+const (
+	ctxStoragePool                 = "storage-pool"
+	ctxStorageVolumeRemoteNodeInfo = "storage-volume-forwarding-node-info"
+)
+
+// addStoragePoolVolumeDetailsToRequestContext extracts details from HTTP requests that pertain to a particular storage
```

Comment:
The comments for these types of function should document what they add to the context, otherwise the review/user has to read the full implementation to understand what it does, which shouldn't be necessary.

Reply:
I've added these details to the function comment now.